pandas/_libs/tslibs/timezones.pyx

```diff
@@ -178,7 +177,7 @@ cdef inline bint is_fixed_offset(object tz):
     return 1
 
 
-cdef object get_utc_trans_times_from_dateutil_tz(object tz):
+cdef object _get_utc_trans_times_from_dateutil_tz(tzinfo tz):
```

Comment:
Is there a reason you deprivatize this? It doesn't seem to be used outside of this module?

Reply:
this is privatizing, not de-privatizing

Reply:
ahum, yeah, ignore my comment :)